compiler/modules/nback/build.bal

```diff
@@ -523,13 +523,13 @@ function buildArithmeticBinary(llvm:Builder builder, Scaffold scaffold, bir:IntA
         builder.positionAtEnd(continueBlock);
         continueBlock = scaffold.addBasicBlock();
         llvm:BasicBlock overflowBlock = scaffold.addBasicBlock();
-        builder.condBr(builder.binaryInt("and",
+        builder.condBr(builder.iBitwise("and",
                                          builder.iCmp("eq", lhs, llvm:constInt(LLVM_INT, int:MIN_VALUE)),
```

Comment:
Please fix the indentation also: the `"and"` and `builder.` should align.

Reply:
Fixed